README.md

```diff
@@ -0,0 +1,65 @@
+# pprx
+
+[![GitHub stars](https://img.shields.io/github/stars/rrwoods/pprx.svg)](https://github.com/rrwoods/pprx/stargazers)
+[![GitHub forks](https://img.shields.io/github/forks/rrwoods/pprx.svg)](https://github.com/rrwoods/pprx/network)
+[![GitHub issues](https://img.shields.io/github/issues/rrwoods/pprx.svg)](https://github.com/rrwoods/pprx/issues)
+
+DDR players who use 3icecream can see their most impressive and most improvable scores!
+
+## Table of Contents
+
+- [Introduction](#introduction)
+- [Features](#features)
+- [Installation](#installation)
+- [Usage](#usage)
+- [Contributing](#contributing)
+- [License](#license)
+- [Acknowledgments](#acknowledgments)
+
+## Introduction
+
+The `pprx` project is designed for DDR (Dance Dance Revolution) players who use the 3icecream platform. This tool helps players to view and analyze their most impressive and most improvable scores.
+
+## Features
+
+- View most impressive scores
+- Identify most improvable scores
+- Easy integration with 3icecream
+
+## Installation
+
+To install `pprx`, clone the repository and install the required dependencies.
+
+```bash
+git clone https://github.com/rrwoods/pprx.git
+cd pprx/pprxweb
+pip install -r requirements.txt
+```
+
+## Usage
+
+To run the tool, use the following command:
+
+```bash
+python manage.py runserver
+```
+
+Follow the on-screen instructions to link your 3icecream account and start analyzing your scores.
```

Comment:
This set of instructions mixes running a PPR X server with PPR X user instructions